source/manage_project/create_new_project.md

```diff
@@ -7,29 +7,16 @@
 1. Run the following in the command line:
 
     ```
-    Middleman init PROJECT_NAME -T alphagov/tech-docs-template
+    middleman init PROJECT_NAME -T alphagov/tech-docs-template
     ```
     where `PROJECT_NAME` is the name of your project.
-    
-1. Answer __Yes__ or __No__ to the installation questions.
 
-1. Project creation is complete when you see `Bundle complete!`.
-    
-## Update your version of the tech doc template
```

Comment:
Does this step no longer apply?

Reply:
See https://github.com/alphagov/tdt-documentation/pull/57/commits/ffc458b8319bbc4ce385204b79bb6c7cb87a7753, it's already covered by a different page.

Reply:
ok